frontend/src/components/models/ModelGroups.css

```
@@ -75,162 +77,255 @@
 
-.model-option input[type="checkbox"] {
+.model-option input[type='checkbox'],
+.model-option input[type='radio'] {
   width: 16px;
   height: 16px;
   cursor: pointer;
-  accent-color: var(--color-burgundy);
+  accent-color: var(--accent);
   flex-shrink: 0;
   -webkit-appearance: checkbox;
   appearance: checkbox;
```

Stylelint (17.13.0)

[error] Expected "checkbox" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)